openapi/Swarm.yaml

```diff
@@ -853,6 +853,9 @@ paths:
   "/addresses":
     get:
       summary: Get overlay and underlay addresses of the node
+      description: This endpoint is available on the main API only if the node is spawned with the `--restricted` flag! And then it requires proper Bearer authentication token.
```

Comment:
nitpick: `This endpoint is available on the main API only if the node is spawned with the `--restricted` flag along with a bearer authentication token.` sounds better.